src/pages/Landing/Testimonials.tsx

```diff
@@ -0,0 +1,178 @@
+import React from 'react';
+import {
+  Container,
+  Box,
+  HStack,
+  VStack,
+  Text,
+  SimpleGrid,
+  useBreakpointValue,
+  Flex,
+  Spacer,
+} from '@chakra-ui/react';
+import { Link } from 'react-router-dom';
+import Carousel, { CarouselItem } from '../../components/Carousel/Carousel';
+import FemaleImage from '../../assets/images/FemaleImage';
+import CardsImage from '../../assets/images/CardsImage';
+import BarsImage from '../../assets/images/BarsImage';
+
+const LandingTestimonials = () => {
+  const baseOrSm = useBreakpointValue({ base: true, md: false });
+  const base = useBreakpointValue({ base: true, sm: false });
+  const CustomStack = baseOrSm ? VStack : HStack;
+  const CustomSignBox = base ? (VStack as typeof Flex) : Flex;
+  return (
+    <Box w="100%">
+      <Container maxW="5xl" centerContent py={20}>
+        <VStack spacing={16} w="100%">
+          <VStack spacing={4}>
+            <Text fontSize="4xl">Here is what users have to say</Text>
+            <Text fontSize="lg" color="gray.500">
+              We had a please of providing this platform of students.
+            </Text>
+          </VStack>
+          <Carousel
+            w="100%"
+            h={{ base: '440px', sm: '375px', md: '200px' }}
+            bgColor="white"
+          >
+            <CarouselItem>
+              <CustomStack>
```

Comment:
i think it would be nice to extract this sub-component you wrote (starting with CustomStack) that is a template content for a CarouselItem from LandingTestimonials, since it is something that we could re-use, so we could include that component in Testimonials instead of rewriting.

Reply:
Refactored.